src/lightning_classify/infer/infer.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, device: torch.device, model_path=None):
                                                             criterion=nn.CrossEntropyLoss(),
                                                             map_location=self.device)
         
-        self.class_dict = self.model.class_dict
+        self.class_dict = torch.load(manifest_path)
```

Comment:
you can just load manifest and directly use it as class dict, you need to use it with key. 

```python
data_dict = torch.load(manifest_path)
data_dict['class_dict']
```